app/graphql/types/coupons/object.rb

```diff
@@ -23,7 +23,9 @@ class Object < Types::BaseObject
       field :expiration_at, GraphQL::Types::ISO8601DateTime, null: true
 
       field :limited_plans, Boolean, null: false
+      field :limited_billable_metrics, Boolean, null: false
```

Comment:
GraphQL/OrderedFields: Fields should be sorted in an alphabetical order within their section. Field `limited_billable_metrics` should appear before `limited_plans`.